tools/cravatool/cravat_submit.xml

```diff
@@ -1,33 +1,117 @@
-<tool id="cravat_submit" name="CRAVAT Submit, Check, and Retrieve" version="0.1.0">
-    <description>Submits, checks for, and retrieves data for cancer annotation</description>
-  <command interpreter="python">cravat_submit.py $input $dropdown $GRCh $psm $Variant $Gene $Noncoding $Error</command>
-  
+<tool id="cravat_submit" name="CRAVAT Submit, Check, and Retrieve" version="0.1.1">
+    <description>| Submits, checks for, and retrieves data for cancer annotation</description>
+  <command interpreter="python">
+  <![CDATA[
+#if $proteo.proteoInput == 'true':
+    #if $analysis.type == 'CHASM' or $analysis.type == 'VEST, CHASM':
+        cravat_submit.py $input $GRCh $Variant $Gene $Noncoding $Error $analysis.type $analysis.classifier $proBED $proteo.intersectedVariants $proteo.output_vcf $vcf_output
+    #else: 
+        cravat_submit.py $input $GRCh $Variant $Gene $Noncoding $Error $analysis.type $proBED $proteo.intersectedVariants $proteo.output_vcf $vcf_output 
+    #end if
+#else:
+    #if $analysis.type == 'CHASM' or $analysis.type == 'VEST, CHASM':
+        cravat_submit.py $input $GRCh $Variant $Gene $Noncoding $Error $analysis.type $analysis.classifier 
+    #else: 
+        cravat_submit.py $input $GRCh $Variant $Gene $Noncoding $Error $analysis.type
+    #end if
+#end if
+]]></command>
   
   <inputs>
   
-    <param format="tabular" name="input" type="data" label="Source file"> </param>
-    <param format="tabular" name="dropdown" type="select" label="Analysis Program">
-      <option value="">None</option>
-      <option value="VEST">VEST</option>
-      <option value="CHASM">CHASM</option>
-      <option value="VEST;CHASM">VEST and CHASM</option>
-    </param>
-    <param format="tabular" name="GRCh" type="select" label="Genome Reference Consortium Human Build (GRCh)">
+    <param format="tabular" name="input" type="data" label="Source file" help="Accepts transcriptomic or genomic inputs (e.g., tabular, VCF). Additional details can be found below."> </param>
+    <conditional name="proteo">
+      <param format="tabular" name="proteoInput" type="boolean" checked="false" label="Include proteogenomic input?" help="Source file (first input) must be in genomic input to enable intersection with this proteogenomic input."></param>
```

Comment:
this should be a select, please read the recommendation here: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?highlight=boolean#booleans